tests/data/expected_avail_processes.json

```diff
@@ -0,0 +1 @@
+{"su": "climdex_days", "id": "climdex_days", "txx": "climdex_mmdmt", "txn": "climdex_mmdmt", "tx10p": "climdex_temp_pctl", "tx90p": "climdex_temp_pctl", "wsdi": "climdex_spells", "fd": "climdex_days", "tr": "climdex_days", "tnx": "climdex_mmdmt", "tnn": "climdex_mmdmt", "tn10p": "climdex_temp_pctl", "tn90p": "climdex_temp_pctl", "csdi": "climdex_spells", "rx1day": "climdex_rxnday", "rx5day": "climdex_rxnday", "sdii": "sdii", "r10mm": "climdex_rmm", "r20mm": "climdex_rmm", "rnnmm": "climdex_rmm", "cdd": "climdex_spells", "cwd": "climdex_spells", "r95ptot": "climdex_ptot", "r99ptot": "climdex_ptot", "prcptot": "climdex_ptot", "gsl": "climdex_gsl", "dtr": "climdex_dtr"}
```

Comment:
Again it may be a good idea to dynamically create this list. That being said I think we can save on some overhead here by added this as a fixture in conftest.